src/qt/bitcoin.cpp

```diff
@@ -430,16 +430,19 @@ int GuiMain(int argc, char* argv[])
 
     BitcoinApplication app(*node);
 
-    // Register meta types used for QMetaObject::invokeMethod
```

Comment:
I don't like how this commit rewrites and reformats this entire section to add a statement. I'd prefer to split this into two commits: one that adds the missing line, and one that's purely restyling / comment updates.

Reply:
Like I said, I also prefer a single line addition.

Reply:
> I'd prefer to split this into two commits: one that adds the missing line, and one that's purely restyling / comment updates.

Done.